iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -128,7 +129,7 @@ private async Task InitializeAndOpenClientAsync()
         // before attempting to initialize or dispose the device client instance.
         private async void ConnectionStatusChangeHandler(ConnectionStatus status, ConnectionStatusChangeReason reason)
         {
-            _logger.LogDebug($"Connection status changed: status={status}, reason={reason}");
+            _logger.LogInformation($"Connection status changed: status={status}, reason={reason}");
```

Comment:
nit: I had chosen debug since I wanted to log disconnection events at a relatively higher log level

Reply:
info is a higher level than debug, right?

Reply:
Yes, you are right. I had mixed up debug and info.